src/components/value/ThreeColumnsValue.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import styled from 'styled-components'
+import background from '@assets/value__bg.svg'
+import mobile from '@assets/value__bg_sm.svg'
+import { TABLET, fontsize } from '@constants/measurements'
+import { Columns as ColumnsBase, Column as ColumnBase, SectionContainerWithBackground as Base } from '@lib/Containers'
+
+const SectionContainer = styled(Base)`
+    @import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@700&family=Fira+Mono:wght@700&display=swap');
```

Comment:
I don't think this is actually doing anything since it is just falling back to monospace! I think that I imported them properly at top level in #58.

Reply:
Thanks for pointing this out! I'll look into this tonight.

Reply:
Just merged `hero` branch and tested locally and seems like it's now correctly picking up! I've taken out this line but once your work gets merged, it should be rendering correctly.